doc/nrf/protocols/bt/bt_mesh/configuring.rst

```diff
@@ -261,7 +261,7 @@ The Bluetooth Mesh security toolbox based on the `PSA Certified Crypto API`_ doe
 After Bluetooth Mesh receives an open key value, it immediately imports the key into the crypto library and receives the unique key identifier.
 The key identifiers are used in the security toolbox and stored in the persistent memory.
 The crypto library is responsible for storing of the key values in the Internal Trusted Storage (`PSA Certified Secure Storage API 1.0`_).
-Bluetooth Mesh data structures based on Tinycrypt and the PSA API, as well as images of these structures stored in the persistent memory, are not compatible due to different key representations.
+Bluetooth Mesh data structures based on Tinycrypt (now removed from the SDK) and the PSA API, as well as images of these structures stored in the persistent memory, are not compatible due to different key representations.
```

Comment:
Nits:
Maybe it should be spelled TinyCrypt.
SDK or |NCS|?

Reply:
@peknis I updated it now.